### PR TITLE
added pdf2video converter

### LINKED ISSUE #1069: Pdf2VideoConverter
# Description
Take a PDF file, extract the text from it, convert it to audio format and attach that audio to the video provided by the user. 

- [x] Python

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
|Pdf2Video Converter | In Description above | feature  | [Sukriti-sood]( https://github.com/Sukriti-sood)|

### PR DESCRIPTION
**Pull request by Sukriti-sood**
# Description

Convert a PDF to Video

Fixes #1069

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works in the Readme.

### CODE REVIEW

**Comment by Sukriti-sood** [comment]
@vybhav72954 Please Review PR.

**Comment by Sukriti-sood** [comment]
@vybhav72954 @HarshCasper  Please Review my PR

File: Python/Pdf2Video_Converter/script.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    language = 'en'
+
+    # Converting text to audio
+    myobj = gTTS(text=mtext, lang=language, slow=False)
+
+    myobj.save("output.mp3")
+
+    audio = MP3("output.mp3")
+
+
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/script.py:62]: remove redundant lines

File: Python/Pdf2Video_Converter/script.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    # Deleting Image files
+    for i in range(1, filelimit + 1):
+        filename = "page_"+str(i)+".jpg"
+        os.remove(filename)
+
+    return mtext
+
+
+def text2video(mtext, video_file, Pdf_file_name):
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/script.py:51]: add docstring to functions

File: Python/Pdf2Video_Converter/script.py (hunk 1 above)
Comment (iamakkkhil) [reply Python/Pdf2Video_Converter/script.py:62]: You can use black to format your code

**Review by iamakkkhil — COMMENTED** [review]
Great work !!

**Review by RohiniRG — APPROVED** [review]
LGTM

File: Python/Pdf2Video_Converter/Readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    or
+    ```
+    python3 script.py
+    ```
+Depending upon the python version. Make sure that you are running the command from the same virtual environment in which the required modules are installed.
+
+
+## Output
+User is asked for a Pdf file and video(mp4) file and Output Video file is stored in the folder.
+
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/Readme.md:26]: It will be great if you will can show us a working of the same

**Review by iamakkkhil — CHANGES_REQUESTED** [review]

File: Python/Pdf2Video_Converter/Readme.md (hunk 3 above)
Comment (Sukriti-sood) [reply Python/Pdf2Video_Converter/Readme.md]: added

File: Python/Pdf2Video_Converter/Readme.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- Run following command :-
+    ```
+    python script.py
+    ```
+    or
+    ```
+    python3 script.py
+    ```
+Depending upon the python version. Make sure that you are running the command from the same virtual environment in which the required modules are installed.
+
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/Readme.md:22]: Remove unnecessary line

File: Python/Pdf2Video_Converter/Readme.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Output
+User is asked for a Pdf file and video(mp4) file and Output Video file is stored in the folder.
+### Sample Output
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/Readme.md:26]: Leave a line above

File: Python/Pdf2Video_Converter/Readme.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+
+https://user-images.githubusercontent.com/55010599/124347215-0c0a1f00-dc01-11eb-9e19-7b36cf78557f.mp4
+
+
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/Readme.md:31]: Remove this line

File: Python/Pdf2Video_Converter/Readme.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    ```
+    python3 script.py
+    ```
+Depending upon the python version. Make sure that you are running the command from the same virtual environment in which the required modules are installed.
+
+
+## Output
+User is asked for a Pdf file and video(mp4) file and Output Video file is stored in the folder.
+### Sample Output
+
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/Readme.md:27]: Remove this line

File: Python/Pdf2Video_Converter/Readme.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+https://user-images.githubusercontent.com/55010599/124347215-0c0a1f00-dc01-11eb-9e19-7b36cf78557f.mp4
```
Comment (iamakkkhil) [inline Python/Pdf2Video_Converter/Readme.md:29]: Use imgur.com as Content delivery network as it is already mentioned in contributing guidelines

**Review by iamakkkhil — CHANGES_REQUESTED** [review]
Make these small small changes

**Pull request closed without merging**